### PR TITLE
Use dropdown for discussion list order input

### LINKED ISSUE #388: Replace the select input on the index with custom dropdown menu
It would be a lot better and consistent for the select input on the index to be replaced by a custom dropdown menu like:
dropdown

### PR DESCRIPTION
**Pull request by datitisev**
Use a dropdown for the discussion list's order select input.
Closes #388 

img
img

### CODE REVIEW

File: js/forum/src/components/IndexPage.js [hunk 1]
```diff
@@ -212,10 +212,21 @@ export default class IndexPage extends Page {
...
+        buttonClassName: "Select-input FormControl",
+        label: sortOptions[this.params().sort] || Object.keys(sortMap).map(key => sortOptions[key])[0],
+        caretIcon: "sort",
+        children: Object.keys(sortOptions).map(value => {
+          const label = sortOptions[value];
+          const active = (this.params().sort || Object.keys(sortMap)[0]) === value;
+
+          return Button.component({
+            children: label,
+            icon: active ? 'check': true,
```
Comment (franzliedke) [inline js/forum/src/components/IndexPage.js]: There's a space missing there. :see_no_evil:

Comment (datitisev) [inline js/forum/src/components/IndexPage.js]: 🙈  Fixed!

**Comment by luceos** [comment]
Well done @datitisev :grin:

**Comment by datitisev** [comment]
Thanks! 😁

**Pull request merged**